client/my-sites/post-type-list/post-actions-ellipsis-menu/view.jsx

```diff
@@ -42,6 +42,7 @@ class PostActionsEllipsisMenuView extends Component {
 
 		this.props.setPreviewUrl( previewUrl );
 		this.props.setLayoutFocus( 'preview' );
+		this.props.onClick();
```

Comment:
> Why isn't the onClick handler from this method being invoked automatically?

Because `PostActionsEllipsisMenuView` never does anything with the `onClick` prop it is being passed. I assume in other instances parent components pass an `onClick` prop to their menu item components, which `PopoverMenu` then can build on. `PostActionsEllipsisMenu` [doesn't pass any props](https://github.com/Automattic/wp-calypso/blob/fcf789ebb43b7d8fe9a912d8fa82d5bee66573b5/client/my-sites/post-type-list/post-actions-ellipsis-menu/index.jsx#L24) though ([with the exception of `globalId` later on](https://github.com/Automattic/wp-calypso/blob/fcf789ebb43b7d8fe9a912d8fa82d5bee66573b5/client/my-sites/post-type-list/post-actions-ellipsis-menu/index.jsx#L44)).

Should I specify default props in `PostActionsEllipsisMenuView` then?